src/main/kotlin/org/rust/cargo/toolchain/tools/Rustfmt.kt

```diff
@@ -145,9 +145,9 @@ class Rustfmt(toolchain: RsToolchainBase) : RustupComponent(NAME, toolchain) {
         }
 
         private fun RsProcessExecutionException.showRustfmtError(project: Project) {
-            val message = message.orEmpty().trimEnd('\n')
+            val message = message.orEmpty().trimEnd('\n').replace("\n", "<br>")
             if (message.isNotEmpty()) {
-                project.showBalloon("Rustfmt", message, NotificationType.ERROR, RustfmtEditSettingsAction("Show settings..."))
+                project.showBalloon("Rustfmt", "<html>$message</html>", NotificationType.ERROR, RustfmtEditSettingsAction("Show settings..."))
```

Comment:
Since we want to use html rendering here and we don't control content of `message`, maybe it's worth to make HTML escaping for `message` to avoid some weird rendering because of special symbols?